doc/devel/make_release.rst

```diff
@@ -75,7 +75,7 @@ Release checklist
   ``README`` in the root directory, maybe with ``vim`` ``diffthis`` command.
   Check all the links are still valid.
 
-* Check all the dipy builds are green on the `nipy buildbot`_
+* Check all the DIPY builds are green on the `nipy buildbot`_
 
 * If you have travis-ci_ building set up you might want to push the code in its
```

Comment:
@skoudoro Thanks for giving it a try!

New patch set 412e9fb:

As for the http://nipy.org/dipy/reference_cmd/CombinedWorkflow.html (`combined_workflow_creation.py`) file's strange rendering, I cannot say much; I've gone through it but did not find any error.

As for the files with `.. admonition::`` rendering problems (`combined_workflow_cration.py`, `gradients_spheres.py`), I did not find a plausible explanation.

As for the wrong rendering for `dipy.tracking.life`, I changed to double inverted commas.

As for the links in FAQ, I think I fixed the links for numpy and cython.

Added `travis-ci` to `link-names.inc`. See whether this solves the issue.